clang/test/CodeGenSYCL/kernelname-enum.cpp

```diff
@@ -0,0 +1,119 @@
+// RUN: %clang_cc1 -I %S/Inputs -fsycl -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -fsycl-int-header=%t.h %s -fsyntax-only
+// RUN: FileCheck -input-file=%t.h %s
+
+#include "sycl.hpp"
+
+enum class no_namespace_int : int {
```

Comment:
Is that on purpose that unscoped enums are no tested ?

Reply:
No. I just missed it. I'll add it.

Reply:
Done